src/errors.js

```diff
@@ -29,6 +29,38 @@ export class InviteAbortedError extends Error {
   }
 }
 
+export class ProjectDetailsSendFail extends Error {
+  /** @param {string} [message] */
+  constructor(message = 'Project Details Failed to send') {
```

Comment:
nitpicking but seems like sentence-casing is reasonable for this?
```suggestion
  constructor(message = 'Project details failed to send') {
```